apps/storefront-docz/src/components/figmaImage.jsx

```diff
@@ -41,11 +41,15 @@ const Title = styled(Link)`
 `
 
 const parseUrl = (url) => {
-  const fileId = (/(?<=file\/).*(?=\/)/.exec(url) || [])[0] || ''
-  const nodeId = ((/(?<=node-id=).*/.exec(url) || [])[0] || '').replace(
-    '%3A',
-    '_',
-  )
+  // Can't use lookbehind because of Safari's lack of support
+  //const fileId = (/(?<=file\/).*(?=\/)/.exec(url) || [])[0] || ''
+  //const nodeId = ((/(?<=node-id=).*/.exec(url) || [])[0] || '').replace(
+  //  '%3A',
+  //  '_',
+  //)
+  const splitOnFile = url.split(/(file\/.*?\/)/)
+  const fileId = splitOnFile[1].split(/(\/)/)[2] || ''
```

Comment:
This could potentially give a `undefined` exception of `splitOnFile` if the split doesn't not split